### ISSUE TITLE
Autocomplete with multi = true dropdown arrow click gets error on version >= 2.0.1

### ISSUE THREAD

**Issue by gsaraceno** [primefaces/primeng#2336]
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
On dropdown arrow click, an error will throw

in Chrome:
> Error in ./AutoComplete class AutoComplete - inline template:16:13 caused by: Cannot read property 'nativeElement' of undefined

in Firefox:

> Error: Error in ./AutoComplete class AutoComplete - inline template:16:13 caused by: this.inputEL is undefined

**Expected behavior**
open panel and launch function assigned on completeMethod event

* **Angular version:** 2.0.X
Angular 2.4.10

* **PrimeNG version:** 2.0.X
PrimeNG >= 2.0.1

* **Browser:** all 
 
* **Language:** Typescript 2.2.1

**Comment by daryl0313** [comment]
I'm seeing the same issue too.

- **Angular version:** 2.0.X
Angular 2.2.4

- **PrimeNG version:** 2.0.X
PrimeNG 2.0.2

I dug into the autocomplete code and found that the error is happening when executing `this.inputEL.nativeElement.focus()` in the code below.  The problem is that `this.inputEL` is undefined when multiple=true, but is defined when multiple=false.
```
    onDropdownFocus() {
        this.dropdownFocus = true;
        this.inputEL.nativeElement.focus();
    }
```

The reason that it's not defined when multiple=true is because of the directive for accessing the input element is not correct in this case.  The element is accessed with `@ViewChild('in') inputEL: ElementRef;`.

```
    <input *ngIf="!multiple" #in pInputText ...
    <ul *ngIf="multiple" ...
        ...
        <li class="ui-autocomplete-input-token">
            <input #multiIn type="text" ...
        </li>
    </ul>
```

From the autocomplete template, you can see that the `#in` directive will not work when multiple=false because it should be `#multiIn`.  I think this bug could be fixed if the directive on both inputs were the same, i.e. `#in`.

The only work around I can think of for now is to implement your own dropdown button and handle the events (focus, click, etc.) manually.

**Comment by daryl0313** [comment]
Looks like this may have been fixed with #2391

**Comment by sudheerj** [comment]
Closing the issue as it got fixed. Please comment out for any concerns.
